public/app/features/plugins/plugin_loader.ts

```diff
@@ -118,7 +119,18 @@ export async function importPluginModule({
     return importPluginModuleInSandbox({ pluginId });
   }
 
-  return SystemJS.import(modulePath);
+  return SystemJS.import(modulePath).catch((e) => {
+    let error = new Error('Could not load plugin: ' + e);
+    console.error(error);
+    pluginsLogger.logError(error, {
+      path,
+      pluginId,
+      pluginVersion: version ?? '',
+      expectedHash: moduleHash ?? '',
+      sriChecksEnabled: (config.featureToggles.pluginsSriChecks ?? false).toString(),
+    });
+    throw error;
+  });
```

Comment:
Are we happy to only track usage of  `importPluginModule` or do want to track all usage of `System.import`?

If it's the latter we likely want to extend the System prototype import and move this logging there. If we only care about module.js then we can ignore this comment entirely. 😄